std/datetime/timezone.d

```diff
@@ -3257,3 +3257,1077 @@ else version(Windows)
+        case "SA Pacific Standard Time": return "America/Bogota";
+        case "SA Western Standard Time": return "America/La_Paz";
+        case "SE Asia Standard Time": return "Asia/Bangkok";
+        case "Sakhalin Standard Time": return "Asia/Sakhalin";
+        case "Saint Pierre Standard Time": return "America/Miquelon";
+        case "Samoa Standard Time": return "Pacific/Apia";
+        case "Singapore Standard Time": return "Asia/Singapore";
+        case "South Africa Standard Time": return "Africa/Johannesburg";
+        case "Sri Lanka Standard Time": return "Asia/Colombo";
+        case "Syria Standard Time": return "Asia/Damascus";
+        case "Taipei Standard Time": return "Asia/Taipei";
+        case "Tasmania Standard Time": return "Australia/Hobart";
+        case "Tocantins Standard Time": return "America/Arguaina";
+        case "Tokyo Standard Time": return "Asia/Tokyo";
+        case "Tomsk Standard Time": return "Asia/Tomsk";
+        case "Tonga Standard Time": return "Pacific/Tongatapu";
+        case "Transbaikal Standard Time": return "Asia/Chita";
+        case "Turkey Standard Time": return "Europe/Istanbul";
+        case "Turks And Caicos Standard Time": return "America/Grand_Turk";
+        case "US Eastern Standard Time": return "America/Indianapolis";
+        case "US Mountain Standard Time": return "America/Phoenix";
+        case "UTC": return "Etc/GMT";
+        case "UTC+12": return "Etc/GMT-12";
+        case "UTC-02": return "Etc/GMT+2";
+        case "UTC-08": return "Etc/GMT+8";
+        case "UTC-09": return "Etc/GMT+9";
+        case "UTC-11": return "Etc/GMT+11";
+        case "Ulaanbaatar Standard Time": return "Asia/Ulaanbaatar";
+        case "Venezuela Standard Time": return "America/Caracas";
+        case "Vladivostok Standard Time": return "Asia/Vladivostok";
+        case "W. Australia Standard Time": return "Australia/Perth";
+        case "W. Central Africa Standard Time": return "Africa/Lagos";
+        case "W. Europe Standard Time": return "Europe/Berlin";
+        case "W. Mongolia Standard Time": return "Asia/Hovd";
+        case "West Asia Standard Time": return "Asia/Tashkent";
+        case "West Bank Standard Time": return "Asia/Hebron";
+        case "West Pacific Standard Time": return "Pacific/Port_Moresby";
+        case "Yakutsk Standard Time": return "Asia/Yakutsk";
+        default: return null;
+    }
+}
+
+version(Windows) version(UpdateWindowsTZTranslations) deprecated @system unittest
+{
+    import std.stdio : stderr;
+
+    foreach (winName; WindowsTimeZone.getInstalledTZNames())
+    {
+        if (windowsTZNameToTZDatabaseName(winName) is null)
+            stderr.writeln("Missing Windows to TZName translation: ", winName);
+    }
+}
+
+
+// This script is for regenerating tzDatabaseNameToWindowsTZName and
+// windowsTZNameToTZDatabaseName from
+// http://unicode.org/cldr/data/common/supplemental/windowsZones.xml
```

Comment:
In a next cleanup step, this could be moved to a directory like `/tools`, so that we can ensure that the script will compile with the latest DMD version ;-)